hw07-patterns/src/main/java/ru/otus/edu/levina/atm/impl/cmd/AbstrCmd.java

```diff
@@ -0,0 +1,40 @@
+package ru.otus.edu.levina.atm.impl.cmd;
+
+import java.util.Date;
+import java.util.UUID;
+
+import lombok.Data;
+import ru.otus.edu.levina.atm.api.ATM;
+import ru.otus.edu.levina.atm.api.cmd.CmdResponse;
+import ru.otus.edu.levina.atm.api.cmd.Command;
+
+@Data
+public abstract class AbstrCmd implements Command {
+
+    protected final String cmdId;
+    protected final Date created;
+    
+    public AbstrCmd() {
+        cmdId = genCmdId();
+        created = new Date();
+    }
+
+    protected String genCmdId() {
```

Comment:
```suggestion
    private String genCmdId() {
```

Лучше сделать приватным. 
`cmdId` храниться здесь, а наследники еще могут чего-нибудь поломать через использование этого метода.